test/Microsoft.Azure.ServiceBus.UnitTests/Management/ManagementClientTests.cs

```diff
@@ -605,6 +605,14 @@ public async Task CorrelationFilterPropertiesTest()
             await client.DeleteTopicAsync(topicName);
         }
 
+        [Fact]
+        [DisplayTestMethodName]
+        public async Task GetNamespaceInfoTest()
+        {
+            var nsInfo = await client.GetNamespaceInfoAsync();
+            Assert.NotNull(nsInfo);
```

Comment:
Should this be a test that asserts the retrieved information rather than just checking for null? I.e. `nsInfo.MessagingSku == MessagingSku.Standard`.
Now that not be the case if someone running it locally with a premium tier, but I'd not worry about that case too much. Most of the test executions are happening on CI where namespace type is always the same.

Reply:
I did think about that. I couldnt come to a conclusion so left it as-is. I can do that Assert

Reply:
👍 null could be misleading as the object would be there, but none of the right values would be in place 🙂